src/mpi/init/finalize.c

```diff
@@ -159,6 +159,7 @@ int MPI_Finalize(void)
 
     MPII_hw_topo_finalize();    /* to be eventually replaced by MPII_hwtopo_finalize() */
```

Comment:
Which commit eventually removes this line?

Reply:
I think I have removed these in the commit before the last. I should move these to the last commit.